### ISSUE TITLE
Firewall page is not accessible

### ISSUE THREAD

**Issue by andreasn** [cockpit-project/cockpit#12940]
Followup for PR #12884

The Firewall page has some general accessibility issues that I found using the Axe Firefox extension. It's especially bad if you use a screen reader.
I want to learn how to make web sites more accessible, so I would be happy to work on this.

**Comment by andreasn** [comment]
@jgiardino sent me this: http://patternfly-a11y-training.surge.sh/

**Comment by andreasn** [comment]
Some comments out of #12884
* The delete buttons will be unreadable to a screen reader. Needs arialabel attached to the button element at a minimum and ideally if we're able to generate that somehow.
* Add Service buttons doesn't say what it is adding a service to. So maybe some aria-label saying "add service: Fedora Workstation Zone" etc.
* The ON/OFF switch misses a label that says what it's turning on or off.

**Comment by Gundersanne** [comment]
@andreasn  have all these things been fixed in #13092 ?

**Comment by andreasn** [comment]
Yes!
